### PR TITLE
Add discourser badge

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: app/images/icons/chatterbox.svg
Comment (ErikSchierboom) [inline app/images/icons/chatterbox.svg:1]: This needs to be changed

File: app/models/badges/chatterbox_badge.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module Badges
+  class ChatterboxBadge < Badge
+    seed "Chatterbox",
+      :common,
```
Comment (ErikSchierboom) [inline app/models/badges/chatterbox_badge.rb]: I've made this common, but I found it hard to estimate

File: app/models/badges/chatterbox_badge.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      'chatterbox',
+      'Linked your Exercism account to Discord'
+
+    def award_to?(user) = has_discourse_account?(user)
+
+    def send_email_on_acquisition? = false
```
Comment (ErikSchierboom) [inline app/models/badges/chatterbox_badge.rb]: I've set this to `false` for now, but 🤷

File: app/models/badges/discourser_badge.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+module Badges
+  class DiscourserBadge < Badge
+    seed "Discourser",
+      :common,
+      'discourser',
+      'Linked your Exercism account to the Forum'
```
Comment (iHiD) [inline app/models/badges/discourser_badge.rb]:
```suggestion
      'Joined Exercism's forum'
```

**Review by iHiD — APPROVED** [review]

**Pull request merged**